### PR TITLE
Support ringcode and PIC for triple/quad-layer 

### PR DESCRIPTION
**Pull request by fuzz6001**
* Output ringcode information of layer 2 and 3 to !submissionInfo.txt
* Increase PIC length

# Ringocde Information samples
## triple-layer
```
Layer 0 (Outer) Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 0 (Outer) Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 0 (Outer) Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
Data Side Mould SID Code: (REQUIRED, IF EXISTS)
Data Side Additional Mould: (REQUIRED, IF EXISTS)
Layer 1 Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 1 Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 1 Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
Label Side Mould SID Code: (REQUIRED, IF EXISTS)
Layer 2 (Inner) Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 2 (Inner) Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 2 (Inner) Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
```

## quad-layer
```
Layer 0 (Outer) Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 0 (Outer) Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 0 (Outer) Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
Data Side Mould SID Code: (REQUIRED, IF EXISTS)
Data Side Additional Mould: (REQUIRED, IF EXISTS)
Layer 1 Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 1 Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 1 Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
Label Side Mould SID Code: (REQUIRED, IF EXISTS)
Layer 2 Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 2 Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 2 Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
Layer 3 (Inner) Mastering Code (laser branded/etched): (REQUIRED, IF EXISTS)
Layer 3 (Inner) Mastering SID Code: (REQUIRED, IF EXISTS)
Layer 3 (Inner) Toolstamp or Mastering Code (engraved/stamped): (REQUIRED, IF EXISTS)
```

### CODE REVIEW

File: MPF.Core/Modules/DiscImageCreator/Parameters.cs [hunk 1]
```diff
@@ -496,7 +496,18 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, Options options
...
+                                    trimLength = 520;
+                                }
+                                else if (info.SizeAndChecksums!.Layerbreak2 != default)
+                                {
+                                    trimLength = 392;
+                                }
+                                else
+                                {
+                                    trimLength = 264;
+                                }
```
Comment (mnadareski) [inline MPF.Core/Modules/DiscImageCreator/Parameters.cs:510]: Minor, but for each of these if/else trees, can you remove the `{` and `}`?

File: MPF.Core/Modules/DiscImageCreator/Parameters.cs [hunk 2]
```diff
@@ -496,7 +496,18 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, Options options
                             case RedumpSystem.SonyPlayStation3:
                             case RedumpSystem.SonyPlayStation4:
                             case RedumpSystem.SonyPlayStation5:
-                                trimLength = 264;
+                                if (info.SizeAndChecksums!.Layerbreak3 != default)
```
Comment (mnadareski) [inline MPF.Core/Modules/DiscImageCreator/Parameters.cs:499]: Does this need to be done for either Aaru or Redumper as well? I'd expect Redumper may benefit from at least part of this.

Comment (fuzz6001) [reply MPF.Core/Modules/DiscImageCreator/Parameters.cs:499]: Sorry, but I can't understand the question.
I do not know much about Aaru.

File: MPF.Core/Modules/DiscImageCreator/Parameters.cs (hunk 1 above)
Comment (fuzz6001) [reply MPF.Core/Modules/DiscImageCreator/Parameters.cs]: fixed (it is my practice)

File: MPF.Core/Modules/DiscImageCreator/Parameters.cs (hunk 2 above)
Comment (Deterous) [reply MPF.Core/Modules/DiscImageCreator/Parameters.cs:499]: @mnadareski this PR already does this for Redumper.
and Aaru/Parameters.cs does not currently parse Layerbreak for blurays anyway.

Comment (mnadareski) [reply MPF.Core/Modules/DiscImageCreator/Parameters.cs:499]: I paid attention, I swear. That was what I was looking for and then promptly overlooked it.

**Pull request merged**